lab-lindsay/test/test.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const assert = require('assert');
+const greet = require('../lib/greet.js');
+
+describe('greet.js', function() {
+  describe('#greetName', function() {
```

Comment:
Using the # mark is good practice to get used to. It's used to indicate a method instantiation. In this example, .greetName is being called, and can be differentiated from other iterations. It will be easier to understand if you look at production level code and see the same method being called in different modules.